site/.htaccess

```diff
@@ -1,12 +1,13 @@
 RewriteEngine On
 
-## Resrtict access to known users ##
+## Restrict access to known users or docker development ##
 AuthType Basic
 AuthName "Restricted Site: Crowdcast"
 AuthBasicProvider file
 AuthUserFile /var/www/passwords
 Require user cctest
-## Resrtict access to known users ##
+Require ip 172.16.0.0/12
```

Comment:
Is this supposed to be **127**.16.0.0/12?

Reply:
thanks for taking a close look! that's a great question.

**172** is correct in this case (but wow that _is_ similar to 127...)

`172.16.0.0/12` is one of the [private IPv4 address ranges](https://en.wikipedia.org/wiki/Private_network#Private_IPv4_addresses), which happens to be the range that [Docker uses for internal networking](https://docs.docker.com/network/network-tutorial-standalone/).

The idea here is that we either want a valid user _or_ someone connecting from a local private network, particularly Docker's network.

Reply:
Ah -- the commit message says 127. probably muscle memory :)

Reply:
oof, definitely muscle memory. glad i typo'd the commit message and not the htaccess file :smile: